src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -229,19 +231,24 @@ class EntityService(rawlsDAO: RawlsDAO, importServiceDAO: ImportServiceDAO, cwds
 
     val importRequest = AsyncImportRequest(gcsPath, FILETYPE_RAWLS, Some(ImportOptions(None, Some(isUpsert))))
 
-    if (cwdsDAO.isEnabled && cwdsDAO.getSupportedFormats.contains(importRequest.filetype.toLowerCase)) {
+    if (useCWDS) {
+      importToCWDS(workspaceNamespace, workspaceName, userInfo, importRequest)
+    } else {
+      importServiceDAO.importJob(workspaceNamespace, workspaceName, importRequest, isUpsert)(userInfo)
+    }
+  }
+
+  private def importToCWDS(workspaceNamespace: String, workspaceName: String, userInfo: UserInfo, importRequest: AsyncImportRequest
```

Comment:
I added in this quick little refactor to avoid duplicated code (had meant to do it in a previous PR and forgot).  Please double-check that I haven't regressed anything here.